tests/test_misc.py

```diff
@@ -461,6 +461,7 @@ def is_p2wpkh(output):
     assert only_one(fundingtx['vin'])['txid'] == res['wallettxid']
 
 
+@unittest.skipIf(not TEST_NETWORK == 'regtest', 'no support for PSETv0')
```

Comment:
Prefer the test expressed as the others" `@unittest.skipIf(TEST_NETWORK == 'liquid-regtest'`